data_warehouse/services/aggregate_data_service.py

```diff
@@ -148,8 +149,8 @@ def records_statistics(cls, context):
     def get_group_records(cls, context):
         '''get group records data'''
         group_by = context.get('group_by', '')
-        start_year = context.get('start_year', 2016)
-        end_year = context.get('end_year', 2016)
+        start_year = context.get('start_year', datetime.now().year)
```

Comment:
`start_year` should be a int if no `start_year` in `context`, but in the later if condition, it's implicitly typed as a string, please fix.